### ISSUE TITLE
GVMEcoder.hh offset [4, 27] is out of the bounds [0, 4]

### ISSUE THREAD

**Issue by ZikZakFR** [fuzziqersoftware/newserv#130]
make returns an error when compiling under Archlinux.

````[ 40%] Building CXX object CMakeFiles/newserv.dir/src/GVMEncoder.cc.o
In file included from newserv/src/GVMEncoder.hh:7,
                 from newserv/src/GVMEncoder.cc:1:
In member function ‘parray<ItemT, Count>& parray<ItemT, Count>::operator=(const ItemT*) [with ItemT = char; long unsigned int Count = 28]’,
    inlined from ‘std::string encode_gvm(const Image&, GVRDataFormat)’ at newserv/src/GVMEncoder.cc:81:21:
newserv/src/Text.hh:317:22: error: ‘void* __builtin_memcpy(void*, const void*, long unsigned int)’ forming offset [4, 27] is out of the bounds [0, 4] [-Werror=array-bounds=]
  317 |       this->items[x] = s[x];
      |       ~~~~~~~~~~~~~~~^~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/newserv.dir/build.make:426: CMakeFiles/newserv.dir/src/GVMEncoder.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/newserv.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

**Comment by fuzziqersoftware** [comment]
This should be fixed by d488ccd1.

**Comment by ZikZakFR** [comment]
> This should be fixed by [d488ccd](https://github.com/fuzziqersoftware/newserv/commit/d488ccd1009e8362cef4f8affb7b81171601f807).

I confirm that GVMEncoder isn't reporting any failure anymore, although something else does now.
I'll check if it is reported, if it isn't then I'll open a new issue.
